src/containers/flows/Raffle/index.tsx

```diff
@@ -122,7 +122,7 @@ const RaffleContainer = () => {
 		} else if (hasRaffleStarted) {
 			return (
 				<>
-					Community Presale Mintlist Signup Period Ending in{' '}
+					Mintlist Signup Period Ending in{' '}
```

Comment:
`Mintlist signup period ending in` would be better capitalisation.

Reply:
We don't have mintlist now.